lib/RandomLib/AbstractMcryptMixer.php

```diff
@@ -64,6 +64,11 @@
      */
     public static function test()
     {
+        if (version_compare(PHP_VERSION, '7.1', '>=')) {
```

Comment:
Using `7.1.0` will fail on `RC` releases. Using `7.1` to include alpha, beta, rc, etc.